app/views/Settings/index.tsx

```diff
@@ -169,6 +180,7 @@ const SettingsScreen = ({ navigation }: SettingsScreenProps): JSX.Element => {
             label={t('screen_titles.about')}
             onPress={navigateTo(Screens.About)}
           />
+          <Divider />
```

Comment:
what are your thought on not using `Divider` components? could we handle the layout using flex and padding or margin?

Reply:
This is specifically for drawing the dividing line (really a `SeperatorComponeent` in FlatList terminology I guess?)
We were previously doing this with implicit border colors, but that relies on the margins of the border being the same as intended divider, which isn't true here. 

IMO the real solution is that the `<Section>` component maps elements with diving lines between them, but that feels overkill considering this happens in just this 1 place right now.